packages/cfpb-atomic-component/src/utilities/dom-traverse.js

```diff
@@ -15,34 +15,53 @@ function queryOne( expr, con ) {
 /**
  * Get the nearest parent node of an element.
```

Comment:
Can you mention in the JSDoc comment that traversal begins with the element and it'll return itself if applicable?